documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,113 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application. You can change how this is configured and make use of HTTP Boot for your own purposes.
+
+=== Requirements
+
+To use HTTP boot you need to use the LATEST / STABLE bootloader configuration, and update to the bootloader dated on or after 10 March 2022. HTTP boot only works over ethernet, so you need an Ethernet cable connected to the network, e.g. via a socket on the back of your router.
```

Comment:
"update to the bootloader dated on or after 10 March 2022." -> "update to a bootloader dated 10th March 2022 or later." ?